interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1480,9 +1480,12 @@ def __init__(self, name='', elements='', species='', note='',
         self.kinetics = None if kinetics == "None" else kinetics
         self.transport = None if transport == "None" else transport
         self.site_density = site_density
+        self.adjacent_phases = phases.split()
 
     def get_yaml(self, out):
         super().get_yaml(out)
+        if self.adjacent_phases:
+            out['adjacent-phases'] = FlowList(self.adjacent_phases)
```

Comment:
Is this covered by a unit test?

Reply:
This one and the route via `ctml2yaml` are covered now as well.